Rules.py

```diff
@@ -257,9 +251,9 @@ def global_rules(world):
     set_rule(world.get_entrance('Clock Town East Gate'), lambda state: state.form('Human') or state.form('Goron') or state.form('Zora') or state.can('Clock Town Guard Skip'))
 
     ### TERMINA FIELD
-    set_rule(world.get_entrance('Astral Observatory Fence'), lambda state: (state.has('Magic Beans') and state.has('Bottle')) or state.can('Goron Boost'))
+    set_rule(world.get_entrance('Astral Observatory Fence'), lambda state: (state.has('Magic Beans') and state.has_bottle()) or state.can('Goron Boost'))
 
-    set_rule(world.get_entrance('TF to Obs Over Fence'), lambda state: state.can('Goron Boost'))
+    set_rule(world.get_entrance('TF to Obs Over Fence Maybe'), lambda state: state.can('Goron Boost'))
```

Comment:
This also needs TF to have everything spawed in (which is an indirect requirement of the Ocarina)